traefik/templates/_podtemplate.tpl

```diff
@@ -399,6 +404,11 @@
           {{- end }}
           {{- end }}
           {{- end }}
+          {{- if .Values.hub.enabled }}
+          - "--experimental.hub"
+          - "--hub"
+          - "--entrypoints.traefikhub-tunl.address=:9901/tcp"
```

Comment:
:thinking: Port 9901 is already the default, maybe we can allow modifying via values? 
see also: 
https://github.com/traefik/traefik/blob/9810bde68bcbbd4f2f4f9a0e91d25129c2503346/docs/content/traefik-hub/index.md#traefikhub-tunl

Reply:
same for api port